patches/tModLoader/Terraria/ModLoader/DamageClassData.cs

```diff
@@ -3,13 +3,17 @@
 	internal struct DamageClassData
```

Comment:
Any reason for internal instead of public?
Is there a reason not to allow mods to access members of this struct/instantiate this struct? I imagine several mods would want to be able to read the active player stat modifiers

Reply:
personally I never knew why it was internal, fine with makin' it public if desired

Reply:
Make everything public, this is a modding API.

Reply:
(resolved)